src/scripts/main.js

```diff
@@ -5,4 +5,26 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  if (person.sex === 'f') {
+    person.sex = 'Female';
+  }
+
+  if (person.sex === 'm') {
+    person.sex = 'Male';
+  }
+
+  table.insertAdjacentHTML('beforeend', `
+    <td>${person.name}</td>
+    <td>${person.sex}</td>
```

Comment:
```suggestion
    <td>${(person.sex === 'm') ? 'Male' : 'Female'}</td>
```
Давай не будемо змінювати поля у вхідному об'єкті, а просто відображати необхідне нам слово в залежності від person.sex

Reply:
Исправь этот комментарий